selfdrive/car/gm/interface.py

```diff
@@ -104,6 +104,13 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None, disa
       ret.lateralTuning.pid.kiV = [0.]
       ret.lateralTuning.pid.kf = 1.  # get_steer_feedforward_volt()
       ret.steerActuatorDelay = 0.2
+      
+      ret.longitudinalTuning.kpBP = [5., 15., 35.]
+      ret.longitudinalTuning.kpV = [0.9, 1.0, 0.8]
+      ret.longitudinalTuning.kiBP = [5., 15., 35.]
+      ret.longitudinalTuning.kiV = [0.12, 0.16, 0.13]
+      ret.longitudinalActuatorDelayLowerBound = 0.41
+      ret.longitudinalActuatorDelayUpperBound = 0.41
```

Comment:
Is this necessary?

Reply:
Neglected to respond to this. Yes, the lookup table alone resulted in the existing long tune gains being too high and it didn't smooth stops at all really. The more accurate (essentially) long feedforward provided by the lookup table decreased error so much that the gains could be lowered considerably. If those same lower gains were used without the lookup table, however, then in many instances the car would probably hit the lead for quicker stops.